test/saithrift/tests/sail3.py

```diff
@@ -2025,3 +2025,632 @@ def runTest(self):
+                pkt = simple_tcp_packet(
+                                eth_dst=router_mac,
+                                eth_src=src_mac,
+                                ip_dst=ip_dst,
+                                ip_src=ip_src,
+                                tcp_sport=sport,
+                                tcp_dport=dport,
+                                ip_id=106,
+                                ip_ttl=64)
+                exp_pkt1 = simple_tcp_packet(
+                                eth_dst='00:11:22:33:44:56',
+                                eth_src=router_mac,
+                                ip_dst=ip_dst,
+                                ip_src=ip_src,
+                                tcp_sport=sport,
+                                tcp_dport=dport,
+                                ip_id=106,
+                                ip_ttl=63)
+                exp_pkt2 = simple_tcp_packet(
+                                eth_dst='00:11:22:33:44:57',
+                                eth_src=router_mac,
+                                ip_dst=ip_dst,
+                                ip_src=ip_src,
+                                tcp_sport=sport,
+                                tcp_dport=dport,
+                                ip_id=106,
+                                ip_ttl=63)
+
+                send_packet(self, 2, str(pkt))
+                rcv_idx = verify_any_packet_any_port(self, [exp_pkt1, exp_pkt2], [1, 3])
+                count[rcv_idx] += 1
+                sport += 1
+                dport += 1
+
+            for i in range(0, 2):
+                print "Packet received on port %s  >> %s" % (i, count[i])
+                self.assertTrue((count[i] >= ((max_itrs / 2) * 0.8)),
+                                "Not all paths are equally balanced, %s" % count)
+            print "All paths are  balanced with two members"
+
+            # Ecmp - iii
+            # Remove next hop group members
+
+            self.client.sai_thrift_remove_next_hop_group_member(nhop_gmember2)
+            self.client.sai_thrift_remove_next_hop_group_member(nhop_gmember3)
+            time.sleep(2)
+            # Send traffic from port 3 with destination IP (10.10.10.1/24) and observe the traffic should drop
+            max_itrs = 100
+            for i in range(0, max_itrs):
+                src_mac = src_mac_start + str(i % 99).zfill(2)
+                ip_src = ip_src_start + str(i % 99).zfill(3)
+                ip_dst = ip_dst_start + str(i % 99).zfill(3)
+                pkt = simple_tcp_packet(
+                                eth_dst=router_mac,
+                                eth_src=src_mac,
+                                ip_dst=ip_dst,
+                                ip_src=ip_src,
+                                tcp_sport=sport,
+                                tcp_dport=dport,
+                                ip_id=106,
+                                ip_ttl=64)
+                exp_pkt1 = simple_tcp_packet(
```

Comment:
exp_pkt1/2/3 is not relevant in this case. Can you remove this?

Reply:
Addressed comments